41-vaccine-temp-monitor/README.md

```diff
@@ -0,0 +1,321 @@
+
+Host Firmware Setup
+-------------------
+
+1. Open Visual Studio Code.
+1. Click the PlatformIO icon on the left hand side and open this project's `firmware/arduino` folder with Quick Access > PIO Home > Open > Open Project.
+1. Open the file `src/main.cpp`.
+1. Uncomment the following line:
+
+   ```c
+   #define PRODUCT_UID "com.your-company:your-product-name"
+   ```
+
+1. Replace `com.your-company:your-product-name` with your ProductUID generated
+   during [Notehub Setup](https://dev.blues.io/notehub/notehub-walkthrough/).
+1. Click the PlatformIO icon on the left hand side again and click "Build" under
+**Project Tasks > featheresp32 > General**.
+
+   You should see "SUCCESS" in the terminal output pane. The firmware image is
+   now ready to be flashed to the host.
+
+1. Click "Upload" under **Project Tasks > featheresp32 > General**.
+
+   You should see "SUCCESS" in the terminal output pane.
+
+Environment Variables
+---------------------
+
+- `alarm_temperature` - The temperature at which the alarm should trigger.
+- `alarm_hysteresis` - The temperature at which the alarm should reset.
+- `sample_interval_s` - The temperature and battery sampling interval in seconds.
+
+_**NOTE:** If you modify any environment variables, you must wait until the next
+synchronization event (max wait: inbound frequency) for the changes to propagate
+to the host._
+
+Operation
+---------
+
+At this point, the host firmware should be running and you should start to see
+events show up on Notehub under your project's "Events" view.
+
+The primary Notefile associated with the application is `status.qo`.
+
+### Normal Operation
+
+#### Temperature Sampling
+
+By default, the host will sample time stamped temperature data from the LM75A to
+`status.qo` every 30 minutes. You can change this interval by setting the
+environment variable `sample_interval_s`. Regardless of the sampling interval,
+the host will publish the collected data to Notehub based on the amount of
+charge remaining on the Li-Ion battery (depicted in table below).
+
+| Charge Level | Outbound Freq (min) | Inbound Freq (min) |
+|:------------:|:-------------------:|:------------------:|
+|   ON POWER   |           5         |          5         |
+|     HIGH     |          60         |        120         |
+|    NORMAL    |         120         |        240         |
+|     LOW      |         240         |        480         |
+|     DEAD     |           0         |          0         |
+
+_**NOTE:** Reducing `sample_interval_s` will cause the application to sample the
+temperature and battery condition more frequently and will shorten the expected
+battery life of the device._
+
+Normal data:
+
+```json
+{
+    "temp":18.5,
+    "voltage":3.5898438
+}
+```
+
+#### Location sampling
+
+The device is configured to use motion based GPS tracking. When motion is
+detected, the device will report its location periodically; every 15 minutes.
```

Comment:
I find this sentence hard to understand. Does it report when motion is detected, or every 15 minutes? How can it be both? Maybe something like this would be better:

As long as motion has been detected, the device will report its location once every 15 minutes.

Reply:
Tried to clean it up by shuffling words around. Let me know what you think.